src/js/stores/library.js

```diff
@@ -196,9 +198,19 @@ define(function (require, exports, module) {
             this._libraryCollection = payload.collection;
             this._updateLibraries(payload.lastSelectedLibraryID);
             
-            var editStatusArray = _.map(payload.editStatus, function (status, documentID) {
-                return [parseInt(documentID), status];
-            });
+            var editStatusArray = _.reduce(payload.editStatus, function (result, status, documentID) {
+                // Remove orphan edit status that are not associate with any open documents.
+                
+                var document = this.flux.stores.document.getDocument(documentID),
+                    documentName = document ? document.name : "";
+                
+                if (documentName.indexOf(status.elementID) !== -1) {
+                    result.push([parseInt(documentID), status]);
```

Comment:
I'm not sure why documentID is a string here? Can you just use `document.id` instead? In any case, you shouldn't use `parseInt` (and _especially_ not without a radix) because it returns `NaN` if it fails to parse. Instead, if you have to parse the string, use the helper in `js/util/math` and test for null.